src/vs/workbench/contrib/webviewPanel/browser/webviewWorkbenchService.ts

```diff
@@ -177,15 +177,15 @@ export class WebviewEditorService extends Disposable implements IWebviewWorkbenc
 		this._iconManager = this._register(this._instantiationService.createInstance(WebviewIconManager));
 
 		this._register(_editorService.onDidActiveEditorChange(() => {
-			this.updateActiveWebview();
+			this.updateActiveWebview(true);
```

Comment:
`isFocused == true` doesn't make sense to me here. I think the parameter either needs to be renamed or we should fix this another way